json-lint/schemas/inventory/adservers/outbrain.json

```diff
@@ -58,6 +58,10 @@
         "pattern": "^(\\$\\{.+\\})$",
         "title": "integer",
         "description": "example: 250 (not required, default in outbrain.tag(100))"
+      },
+      "disableMarfeelStyles": {
```

Comment:
2 things:
1) It should be `disableCustomStyles`, check the implementation of Outbrain in XP
2) It's an object parameter in the shape `{ disableCustomStyles: true }`